src/translations/screens/Tickets.ts

```diff
@@ -7,7 +7,7 @@ const TicketsTexts = {
   header: {
     title: _('Billetter', 'Tickets'),
   },
-  buyTicketsTab: {
+  purchaseTab: {
```

Comment:
Should we also change main object TicketsTexts to something else here? This and the other TicketTexts (now TicketingText) has always confused me 😅